pkg/adapter/awssqssource/adapter_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2019 TriggerMesh, Inc
+Copyright (c) 2020 TriggerMesh Inc.
```

Comment:
nitpick: 
```suggestion
Copyright (c) 2019-2020 TriggerMesh Inc.
```

Reply:
Will create a follow up task for it because we'll need to update them all.

Reply:
fair enough!

Reply:
Done! #103